File: 0026-remove-duplicates-from-sorted-array/26.remove-duplicates-from-sorted-array.cs
/*
 * @lc app=leetcode id=26 lang=csharp
 *
 * [26] Remove Duplicates from Sorted Array
 */

// @lc code=start
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        int i = 1;
        for (int j = 1; j < nums.Length; j++)
        {
            if (nums[j] > nums[j-1])
            {
                nums[i++] = nums[j];
            }
        }

        return i;
    }
}
// @lc code=end

File: 0026-remove-duplicates-from-sorted-array/26.remove-duplicates-from-sorted-array.py
#
# @lc app=leetcode id=26 lang=python3
#
# [26] Remove Duplicates from Sorted Array
#

# @lc code=start
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        x, y = 0, 1
        while y < len(nums):
            if nums[x] == nums[y]:
                nums.pop(y)
            else:
                x += 1
                y += 1

        return len(nums)
# @lc code=end

